composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "devdojo/chatter",
```

Comment:
Please do not change the package name.

Reply:
this is because I use it on my site and when I pull, everything gets pull, but you can select not to merge that right?